docs/mobile-apps/automated-testing/appium/quickstart.md

```diff
@@ -62,5 +62,9 @@ mvn clean test
 ```
 Alternatively, you can right click the test and choose `Run` from most IDE interfaces.
 
+:::note
+For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not supported in emulator tests.
```

Comment:
should we maybe say here:
"... as the latter is not supported in emulator test, yet." ?